torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3235,8 +3323,58 @@ def reference_lgamma(x):
     ]
     op_db = op_db + op_db_scipy_reference
 
+op_db += [
+    _BinaryUfuncInfo('add',
+                     aliases=('add',),
+                     ref=np.add,
+                     dtypes=all_types_and_complex(),
+                     dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.half),
+                     test_inplace_grad=True,
+                     supports_out=True,
+                     sample_inputs_func=sample_inputs_binary,
+                     decorators=(precisionOverride({torch.bfloat16: 5e-1,
+                                                    torch.float16: 5e-1}),),
+                     ),
+    _BinaryUfuncInfo('mul',
+                     aliases=('multiply',),
+                     ref=np.multiply,
+                     dtypes=all_types_and_complex(),
+                     dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.half),
+                     test_inplace_grad=True,
+                     supports_out=True,
+                     sample_inputs_func=sample_inputs_binary,
+                     decorators=(precisionOverride({torch.bfloat16: 5e-1,
+                                                    torch.float16: 5e-1}),),
+                     ),
+    _BinaryUfuncInfo('sub',
+                     aliases=('subtract',),
+                     ref=np.subtract,
+                     # skip subtraction of bool types
+                     dtypes=all_types_and_complex(),
+                     dtypesIfCUDA=all_types_and_complex_and(torch.half),
+                     test_inplace_grad=True,
+                     supports_out=True,
+                     sample_inputs_func=sample_inputs_binary,
+                     decorators=(precisionOverride({torch.bfloat16: 5e-1,
+                                                    torch.float16: 5e-1}),),
+                     ),
+    _BinaryUfuncInfo('div',
```

Comment:
I think this PR wants to modify the existing div OpInfos to be _BinaryUfuncInfos, and not create a new one?

Reply:
yes, will do